document.js

```diff
@@ -0,0 +1,30 @@
+function document(body, map){
+return `
+    <html lang="en">
+    <head>
+        <meta charSet="UTF-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        
+       
+<script src='https://api.mapbox.com/mapbox-gl-js/v2.2.0/mapbox-gl.js'></script>
+<link href='https://api.mapbox.com/mapbox-gl-js/v2.1.1/mapbox-gl.css' rel='stylesheet' />
+<link href='/css/style.css' rel='stylesheet' ></link>
```

Comment:
The `<link>` element is known as a "void" element, which it has no closing tag so you can remove `</link>`. Other void elements include `<meta>` and `<img>` and `<br>` (a line break.)